CorinneTest/concreteProductPerson.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
are we using this?

Reply:
no - i thought i had deleted it earlier but really just removed it from the project